packages/inferno-component/src/index.ts

```diff
@@ -44,6 +44,14 @@ export interface ComponentLifecycle<P, S> {
 	componentWillUnmount?: () => void;
 }
 
+export type SFC<P> = StatelessComponent<P>;
```

Comment:
While it's more verbose. In terms of DX I think that only using `StatelessComponent<P>` would be better. I think it will avoid a lot of questions with "What's SFC mean?" for new contributors.